sklearn/svm/tests/test_svm.py

```diff
@@ -844,9 +844,51 @@ def test_svc_bad_kernel():
 
 
 def test_timeout():
-    a = svm.SVC(kernel=lambda x, y: np.dot(x, y.T), probability=True,
+    svc = svm.SVC(kernel=lambda x, y: np.dot(x, y.T), probability=True,
```

Comment:
In general try to avoid changing something which is unrelated to the PR, it adds noise and makes reviewing less pleasant.